### PR TITLE
Set common setting to yaml file

### PR DESCRIPTION
**Pull request by tinawang123**
Set some common settings include ORIGIN_SYSTEM_VERSION
and UPGRADE_TARGET_VERSION to yaml file, in case they can't be recognized.

- Related ticket: https://progress.opensuse.org/issues/102488
- Verification run: https://openqa.suse.de/tests/7697729#

### CODE REVIEW

**Comment by rfan1** [comment]
setting are -> settings are

File: schedule/migration/offline_spvm_Upgrade.yaml [hunk 1]
```diff
@@ -1,6 +1,9 @@
 name:   offline_spvm_Upgrade
 description:    >
     This is for spvm upgrade migration
+vars:
+  ORIGIN_SYSTEM_VERSION: '%HDDVERSION%'
```
Comment (rfan1) [inline schedule/migration/offline_spvm_Upgrade.yaml:5]: 2 blank space or 4 here?

Comment (tinawang123) [reply schedule/migration/offline_spvm_Upgrade.yaml:5]: Two space

**Review by chcao — APPROVED** [review]
LGTM

**Review by lemon-suse — APPROVED** [review]

**Comment by lemon-suse** [comment]
To avoid misunderstanding, I think you can update PR description as :
Set some common settings include ORIGIN_SYSTEM_VERSION
and UPGRADE_TARGET_VERSION to yaml file, in case they can't be recognized.

**Review by rfan1 — APPROVED** [review]
LGTM

**Pull request merged**